packages/web-components/package.json

```diff
@@ -39,8 +39,10 @@
     "lint:src": "yarn lint:scripts",
     "lint:scripts": "eslint --ext .js,.ts,.tsx .",
     "typecheck": "tsc --noEmit -p tsconfig.json",
-    "test": "yarn test:unit",
+    "test": "yarn test:unit && yarn test:integration",
     "test:unit": "gulp test:unit",
+    "test:integration": "yarn yarn test:integration:ui",
```

Comment:
Is yarn supposed to be repeated?

Reply:
Good catch @jeffchew thanks - Updated: https://github.com/carbon-design-system/ibm-dotcom-library/pull/3204/commits/8ba83ea27058c4392c0d8437afef952aea2d5bad